packages/tooling/fast-tooling/src/data-utilities/mapping.spec.ts

```diff
@@ -1062,22 +1062,27 @@ describe("mapWebComponentDefinitionToJSONSchema", () => {
     });
     test("should map attributes and slots to the JSON schema", () => {
         const name: string = "foo";
+        const title: string = "FooBar";
         const description: string = "foo tag";
         const attrName: string = "attr";
+        const attrTitle: string = "attr title";
```

Comment:
"attrName" and "attrTitle" is the only abbreviation.  Can this be spelled out for better readability and consistency?

Reply:
This was present in file previously, if you'd like to create an issue for it we can put it in the backlog but I don't want to muddy the waters on the specificity for this particular change with test file var name updates.